.env.sample

```diff
@@ -1 +1,2 @@
-HOCKEYAPP_APP_ID_STAGING =
+HOCKEYAPP_APP_ID_STAGING = example
+HOCKEYAPP_APP_ID_PRODUCTION = example
```

Comment:
Why not leave these empty?

Reply:
If it's empty then there is a compile error in autogenerated `CarLensKeys.m`.